src/createrepo_c.c

```diff
@@ -1286,6 +1287,7 @@ main(int argc, char **argv)
     user_data.had_errors        = 0;
     user_data.output_pkg_list   = output_pkg_list;
 
+    g_mutex_init(&(user_data.mutex_nevra_table));
```

Comment:
We have to also `g_mutex_clear()` the `user_data.mutex_nevra_table`, its around [here](https://github.com/rpm-software-management/createrepo_c/blob/master/src/createrepo_c.c#L1452-L1455).

Reply:
@kontura Done